ui/admin/app/templates/scopes/scope/auth-methods/index.hbs

```diff
@@ -62,171 +61,153 @@
+                      @hasChevron={{false}}
                     />
-                  {{/unless}}
-                  {{#if B.data.isPrimary}}
-                    <dd.Interactive
-                      @text={{t
-                        'resources.auth-method.actions.remove-as-primary'
-                      }}
-                      {{on 'click' (route-action 'removeAsPrimary' B.data)}}
-                    />
-                  {{/if}}
-                </Hds::Dropdown>
-              {{/if}}
-            </B.Td>
-          </B.Tr>
-        </:body>
-      </Hds::Table>
-
-    {{else if
-      (has-resource-filter-selections 'scopes.scope.auth-methods' 'type')
-    }}
-
-      <Rose::Layout::Centered>
-        <Rose::Message @title={{t 'titles.no-results'}} as |message|>
-          <message.description>
-            {{t 'descriptions.no-results'}}
-          </message.description>
-        </Rose::Message>
-      </Rose::Layout::Centered>
-
-    {{else}}
-
-      <Rose::Layout::Centered>
-        <Rose::Message
-          @title={{t 'resources.auth-method.messages.welcome.title'}}
-          as |message|
-        >
-          <message.description>
-            {{#if (can 'list model' this.scopeModel collection='auth-methods')}}
-              {{! can list (at least):  show default welcome message}}
-              {{t 'resources.auth-method.messages.welcome.description'}}
-            {{else if
-              (can 'create model' this.scopeModel collection='auth-methods')
+                    {{#unless B.data.isPrimary}}
+                      <dd.Interactive
+                        @text={{t 'resources.auth-method.actions.make-primary'}}
+                        data-test-make-primary-action
+                        {{on 'click' (route-action 'makePrimary' B.data)}}
+                      />
+                    {{/unless}}
+                    {{#if B.data.isPrimary}}
+                      <dd.Interactive
+                        @text={{t
+                          'resources.auth-method.actions.remove-as-primary'
+                        }}
+                        {{on 'click' (route-action 'removeAsPrimary' B.data)}}
+                      />
+                    {{/if}}
+                  </Hds::Dropdown>
+                {{/if}}
+              </B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+        <Rose::Pagination
+          @totalItems={{@model.totalItems}}
+          @currentPage={{this.page}}
+          @currentPageSize={{this.pageSize}}
+        />
+      {{else}}
+        <Hds::ApplicationState data-test-no-auth-method-results as |A|>
+          <A.Header @title={{t 'titles.no-results-found'}} />
+          <A.Body
+            @text={{t
+              (if
+                this.search
+                'descriptions.no-search-results'
+                'descriptions.no-filter-results'
+              )
+              query=this.search
+              resource='auth-methods'
```

Comment:
This also should've been a translation, I think we're doing this in a few places

Reply:
good catch! updated!